src/DynamicForm.vue

```diff
@@ -34,6 +35,7 @@
 
     interface Methods {
         buildValue: (control: DynamicControl) => string | string[] | number | null
+        transformValue(value: string | string[] | number | null, transform: string): any
```

Comment:
Perhaps the built value could be given its own custom type to avoid repetition. Not super necessary though.

Reply:
Good idea, will add that